rero_ils/modules/budgets/serializers.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+# Copyright (C) 2021 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Acquisition receipt serialization."""
+
+from invenio_records_rest.serializers.response import record_responsify
+
+from rero_ils.modules.serializers import JSONSerializer, RecordSchemaJSONV1
+
+
+class BudgetJSONSerializer(JSONSerializer):
```

Comment:
You have several times the same code. Probably this can be done in a common class.